docs/contributing.md

```diff
@@ -0,0 +1,202 @@
+---
+id: contributing
+title: Contributing to the Doc Hub
+slug: /contributing
+---
+
+The Experimenter documentation hub is managed in the [mozilla/experimenter-docs](https://github.com/mozilla/experimenter-docs) repository. You will need a GitHub account to modify an existing document or create a new one.
+
+This page will walk you through how to edit an existing document or create a new one, how to display docs in the sidebar where desired, and how to make these changes in GitHub.
+
+Documents are written in Markdown (`.md`) files using [GitHub-flavored Markdown syntax](https://github.github.com/gfm/) and are compiled into static web pages. [MDX](https://mdxjs.com/) is also supported, meaning you may embed React components in content as needed. Refer to Docusaurus [Markdown features](https://v2.docusaurus.io/docs/markdown-features) for more technical information about these Markdown pages.
+
+<div className="flex">
+<div>
+<b>Notifications</b>
+
+If you would like to subscribe to notifications for this repository, including when someone is requesting doc changes with a pull request that you can review and approve, be sure to "Watch" the repository.
+
+Also, see [configuring notifications](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications) in GitHub.
+
+Alternatively, consider subscribing to this repository in the [GitHub Slack](https://slack.github.com/) app by searching for it under "Apps" inside Slack.
+
+</div>
+   <img src="../img/github-watch.png" alt="GitHub Watch" style={{ width: '200px', marginLeft: '16px' }} />
```

Comment:
@jodyheavener I could've added a class or two here instead of `style={{` but I'm not sure that it'd be worth enforcing "no inline styles!!" in the doc hub. It'd be one more thing for reviewers to have to know to lookout for. Whatchu think?

Reply:
Random notion off the top of my head: Rather than enforce no inline styles, I'd ask if this could have been a class that would come in handy to re-use later. Can also be an optimization to perform later, if a common style comes up often

Reply:
Yeah, it's possible. I'm not sure how often folks will actually be using JSX in MD but I'll go ahead and convert these to classes.

Reply:
OK after converting all these to classes, I like what we've got here much better, so good call. Was probably just me being lazy.